doc/changes/devel/12901.bugfix.rst

```diff
@@ -0,0 +1 @@
+:class:`mne.Report` HDF5 files are now written in ``mode='a`` (append) to allow users to store other data in the HDF5 files, by `Eric Larson`_.
```

Comment:
Forgot about this change, it's pretty straightforward I think -- use `mode="a"` to make sure you can write companion data to the same HDF5 file without wiping it out. The report is saved in the root-level key `mnepython` so it's easy to use `h5io.write_hdf5(fid, whatever, "myrootkey")` if you've done `fid = h5py.File(..., "a")` already.

Reply:
```suggestion
:class:`mne.Report` HDF5 files are now written in ``mode='a'`` (append) to allow users to store other data in the HDF5 files, by `Eric Larson`_.
```